packages/api-docs-builder/ApiBuilders/ComponentApiBuilder.ts

```diff
@@ -91,7 +91,10 @@ export function writePrettifiedFile(
  * why the source includes relative url. We transform them to absolute urls with
  * this method.
  */
-async function computeApiDescription(api: ReactApi, options: { host: string }): Promise<string> {
+export async function computeApiDescription(
+  api: { description: ReactApi['description'] },
```

Comment:
The type is simplified so that it can be used with the hooks generation too. Only the description is used in the function.